Scripts/extras/pull_and_build_from_git.py

```diff
@@ -779,6 +779,19 @@ def check_build_keys(self, keys_to_check):
 
         return False
 
+    def copy_extra_files(self):
+        """
+        Copies any extra files specified in the build config into the destination folder for packaging.
+        """
+        extra_files_to_copy = self.platform_config.get('extra_files_to_copy', [])
```

Comment:
if 'extra_files_to_copy' is a new key, you will need to document it above with the other keys

Reply:
extra_files_to_copy moved to PackageInfo. Comment added as well for the new "extra_files_to_copy" build_config key.